SetupConfigure.cmake

```diff
@@ -181,6 +179,88 @@ if(BUILD_CONFIGURATION STREQUAL "APP-WEB")
 
 endif()
 
+if(BUILD_CONFIGURATION STREQUAL "APP-WEB-WORKER")
```

Comment:
Instead of building one executable target twice with different configurations, isn't it much easier to just have two different executable targets?

Reply:
What I mean is, that ideally you should have to run CMake once to build both targets. So, configure once, `cmake --build` once, and `install` once, and then you should have everything in the right place.
Now, you first have to run CMake with one configuration to build one executable, then with another configuration to build another executable, and then combine everything together manually.